config/audit.js

```diff
@@ -17,6 +17,8 @@ module.exports = merge(base, {
 		gaAlternateId: 'UA-11686022-6',
 		enableSnowplow: true,
 		snowplowUri: 'events.fivetran.com/snowplow/v5qt54ocr2nm',
+		enableHotjar: false,
+		hotjarId: '3071239',
```

Comment:
@christian14b @emuvente I think we should at minimum setup a 2nd site that can be used across all lower environments for testing. I'll check our account for that option.